README.md

```diff
@@ -0,0 +1,147 @@
+# Stackable Docker Images
+
+This repository contains Dockerfiles and scripts to build base images for use within Stackable.
+
+<!-- start:badges: autogenerated by ./update_readme_badges.sh -->
```

Comment:
idea: Should we add this to the pre-commit config so that the generating gets run on every commit to always keep it up to date?

Reply:
Yeah, I thought about it and then just didn't.
I'll add a strict mode on the top so that it fails early (eg: if dependent apps aren't installed).

Reply:
Oh, there is no pre-commit config in this repo. Guess we can add it now

Reply:
There will be a follow-up PR to add pre-commit hooks

Reply:
Alright!